common/consts.go

```diff
@@ -28,9 +28,10 @@ const (
 
 // SwapCreator.sol event signatures
 const (
-	ReadyEventSignature    = "Ready(bytes32)"
-	ClaimedEventSignature  = "Claimed(bytes32,bytes32)"
-	RefundedEventSignature = "Refunded(bytes32,bytes32)"
+	NewSwapFunctionSignature = "newSwap(bytes32,bytes32,address,uint256,uint256,address,uint256,uint256)"
+	ReadyEventSignature      = "Ready(bytes32)"
+	ClaimedEventSignature    = "Claimed(bytes32,bytes32)"
+	RefundedEventSignature   = "Refunded(bytes32,bytes32)"
```

Comment:
These are all available programmatically if we don't want to hard code them. Or we could could have a unit test that fails if their values don't match the current ABI.

```go
func TestABISigs(t *testing.T) {
	for name, event := range SwapCreatorParsedABI.Events {
		t.Logf("%s: %s", name, event.Sig)
	}
	t.Log()
	for name, method := range SwapCreatorParsedABI.Methods {
		t.Logf("%s: %s", name, method.Sig)
	}
}
```